src/message.rs

```diff
@@ -1,38 +1,45 @@
 use node::Node;
 use prefix::{Name, Prefix};
-use section::Section;
 
-/// Network message (RPC).
+/// Network message (RPC),
```

Comment:
why changed to `,` ? 
`.` is still used for `NetworkAction` below

Reply:
Just a typo. Fixed.